modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -219,8 +218,8 @@ public Swift5ClientCodegen() {
         typeMapping.put("float", "Float");
         typeMapping.put("number", "Double");
         typeMapping.put("double", "Double");
-        typeMapping.put("file", "URL");
```

Comment:
@aymanbagabas could you please introduce a flag to control the typeMapping to be URL or Data, depending on this flag?
Because we need to give an option to the users to set the old behaviour if they need it.

Reply:
The problem is that this is a big change that can impact users.
From what we discuss in the issue, the file download is not working, bit the file upload is working as expected, and this can potentially break it, so I think it's a good ideia to give an option to keep the old behaviour in case the users found any issue with this change.